[Installation issue]:

### Does this issue concern a missing requirement?

_No response_

### Missing Requirement

_No response_

### What happened?

Install script likely needs `set -euo pipefail` on top. The exit codes used in the script do not fail the script. Missing config directory creates a directory with the error text


```bash
[builder@zaarch Toutui]$ ls
 CODE_OF_CONDUCT.md   Cargo.lock   LICENSE     SECURITY.md                                        assets                hello_toutui.sh   src
 CONTRIBUTING.md      Cargo.toml   README.md  '[ERROR] Cannot find "builder" config directory.'   config.example.toml   known_bugs.md

```

### Steps to reproduce the issue

1. 

### Toutui version that you try to install

_No response_

### On which OS are you trying to install Toutui?

Arch Linux

### Which install method?

Automatic install

### Error message

_No response_

### Additional Notes

_No response_

--- a/hello_toutui.sh
+++ b/hello_toutui.sh
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 # Install Toutui and dependencies automagically.
 
+set -eo pipefail
+
 main() {
     do_not_run_as_root
 

